components/PomodoroTimer.tsx

```diff
@@ -143,18 +155,17 @@ export function PomodoroTimer() {
             setTime(TIMES.WORK);
             setPlaying(false)
             setKey(key + 1);
-            if (currentSong > playlist.length - 1) {
-                setCurrentSong(0);
-            } else {
-                setCurrentSong(currentSong + 1);
-            }
+            nextSound();
         }
     }
 
     const { height } = useWindowDimensions();
 
     const marginTop = height <= 667 ? 50 : 130;
 
+    const playingStateIcon = playing ? 
+    <Icon name="pause" color="white" fontFamily="Feather" fontSize="2xl" /> :
+    <Icon name="play" color="white" fontFamily="Feather" fontSize="2xl" />;
```

Comment:
Since only the name of the icon is changing, you could use the ternary just inside the `name` props.

```suggestion
```